ibtk/include/ibtk/AppInitializer.h

```diff
@@ -148,12 +152,16 @@ class AppInitializer : public SAMRAI::tbox::DescribedClass
     bool dumpRestartData() const;
 
     /*!
-     * Return the restart dump interval.
+     * Return the restart dump interval. Read in the main_db in the main_db
```

Comment:
It might be better to write `"Main"` or explain what `main_db` is. I think you didn't mean to write `in the main_db` twice.